elyra/cli/pipeline_app.py

```diff
@@ -0,0 +1,188 @@
+#
+# Copyright 2018-2020 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import asyncio
+import click
+import json
+import os
+import requests
+
+from yaspin import yaspin
+from colorama import Fore, Style
+
+from elyra import __version__
+from elyra.pipeline import PipelineParser, PipelineProcessorManager
+from jupyter_server.serverapp import list_running_servers
+
+
+def _validate_pipeline_file(pipeline_file):
+    extension = os.path.splitext(pipeline_file)[1]
+    if extension != '.pipeline':
```

Comment:
Do we need to verify that the file has `.pipeline` as an extension?

Reply:
Mostly to avoid other file extensions like you mentioned above (e.g. ipynb, py, etc)